cache/build.gradle

```diff
@@ -5,6 +5,10 @@ description = 'Apollo Cache'
 dependencies {
     implementation project(':util')
     implementation group: 'com.google.guava', name: 'guava', version: guavaVersion
+    implementation group: 'org.tukaani', name: 'xz', version: '1.8'
+    implementation group: 'it.unimi.dsi', name: 'fastutil', version: '8.3.1'
+    implementation group: 'org.apache.ant', name: 'ant', version: '1.10.7'
+    implementation group: 'com.google.code.gson', name: 'gson', version: '2.8.6'
```

Comment:
I'd recommend using Jackson instead of Gson.

Reply:
Sure, no difference to me.